engineering-decision-records/005-amaru-maintainers-committee.md

```
@@ -21,15 +21,16 @@ In order to manage the decision making on the project and have a compelling prod
 
 Here is the new composition of the Amaru maintainers committee (scope owners of the project):
 
-| Matthias Benkort <br/> <img src="https://github.com/ktorz.png" width=50 height=50 /> <br/> [@KtorZ][] | Santiago Carmuega <br/> <img src="https://github.com/scarmuega.png" width=50 height=50 /> <br/> [@scarmuega][] | Pi Lanningham <br/> <img src="https://github.com/quantumplation.png" width=50 height=50 /> <br/> [@QuantumPlation][] |  Arnaud Bailly <br/> <img src="https://github.com/abailly.png" width=50 height=50 /> <br/> [@abailly][] |
-| --- | --- | --- | --- |
-| Ledger & integration | Networking | Mempool & Middleware | Consensus |
+| Matthias Benkort <br/> <img src="https://github.com/ktorz.png" width=50 height=50 /> <br/> [@KtorZ][] | Santiago Carmuega <br/> <img src="https://github.com/scarmuega.png" width=50 height=50 /> <br/> [@scarmuega][] | Pi Lanningham <br/> <img src="https://github.com/quantumplation.png" width=50 height=50 /> <br/> [@QuantumPlation][] |  Arnaud Bailly <br/> <img src="https://github.com/abailly.png" width=50 height=50 /> <br/> [@abailly][] |  Damien Czapla <br/> <img src="https://avatars.githubusercontent.com/u/161828179?v=4" width=50 height=50 /> <br/> [@Dam-CZ][] |
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)